rj_gameplay/stp/rc.py

```diff
@@ -671,6 +675,13 @@ def their_restart(self) -> bool:
 
         return not self.__our_restart
 
+    @property
+    def game_score(self) -> np.array:
+        """
+        :return: The game score
+        """
+        return self.__game_score  # 0 for our score, 1 for opponent score
```

Comment:
this comment is a bit confusing, can you instead clarify that it returns a 2-element array where the first element is our score and the second is their score? feel free to use your own wording but something to that effect would be helpful, like `:return: Score of each team in format [our_score, their_score]` would be good as well

Reply:
also, put it in the docstring above, not as a comment next to the return
```suggestion
        return self.__game_score
```